src/views/SafetyStock.vue

```diff
@@ -12,7 +12,9 @@
         <ScheduleRuleItem v-if="rules.length" />
 
         <section>
-          <RuleItem v-for="(rule, ruleIndex) in rules" :rule="rule" :ruleIndex="ruleIndex" :key="rule.ruleId" />
+          <ion-reorder-group :disabled="false" @ionItemReorder="reorderingRules = doReorder($event, reorderingRules)">
```

Comment:
We should create a method and update the value for reorderingRules in the method instead of updating it in template